mat.mojo

```diff
@@ -0,0 +1,115 @@
+from math import tan, cos, sin
+
+alias Mat4 = SIMD[DType.float32, 16]
+
+def mat4_zero() -> Mat4:
```

Comment:
`def` has the same semantics as adding a `raises` to your function, so `fn` should be preferred in general for methods that are infallible.